nemo/nemo/core/neural_factory.py

```diff
@@ -805,7 +800,9 @@ def optim_level(self):
 
     @property
     def logger(self):
-        return self._exp_manager.logger
+        warnings.warn("This will be deprecated in future releases. Please use "
```

Comment:
hmmmm, we had that definition already in callbacks.py...

anyway, we will get rid both of them soon, so not really important ;)